cmake/compiler_settings.cmake

```diff
@@ -3,8 +3,8 @@
 
 # Check Clang version.
 if (CMAKE_C_COMPILER_ID MATCHES Clang)
-  if (CMAKE_C_COMPILER_VERSION VERSION_LESS 7 OR CMAKE_C_COMPILER_VERSION
-                                                 VERSION_GREATER 7.99)
+  if (CMAKE_C_COMPILER_VERSION VERSION_LESS 8 OR CMAKE_C_COMPILER_VERSION
+                                                 VERSION_GREATER 8.99)
     message(WARNING "Open Enclave officially supports Clang 7 only, "
```

Comment:
This text should also be updated to `Clang 8` instead of `Clang 7`

Reply:
Fixed.